app/src/sharedTest/java/org/oppia/android/app/walkthrough/WalkthroughActivityTest.kt

```diff
@@ -119,7 +119,7 @@ class WalkthroughActivityTest {
   }
 
   @Test
-  fun testWalkthroughFragment_increaseProgress_onBackPressed_decreaseProgress_progressWorksCorrectly() { // ktlint-disable max-line-length
+  fun testWalkthrough_increaseProgress_onBackPressed_decreaseProgress_progressWorksCorrectly() {
```

Comment:
The outcome here is a bit vague. Perhaps this would be clearer: ``reportedProgressAdvances``. This indicates what's expected to happen: the reported progress should show that the user navigated further through the walkthrough.